kytos/core/interface.py

```diff
@@ -246,8 +246,12 @@ def all_tags_available(self) -> bool:
         """Return True if all tags are avaiable (no tags used),
          False otherwise"""
         with self._tag_lock:
-            return (self.available_tags == self.tag_ranges and
-                    self.special_available_tags == self.special_tags)
+            tags_bool = self.available_tags == self.tag_ranges
```

Comment:
Aldo, I think you can keep less variable states if you were to leverage early returns. For instance:

```
if self.available_tags != self.tag_ranges:
    return False
```

And, similarly in the special_available_tags items loop, just return mid iteration if a difference is found. 

Finally, good point, a `set()` for the values of available_tags seems more suited and lightweight.

Reply:
Applied early returns and changed from `Counter` to `set`.